### PR TITLE
KOGITO-437: Stunner - Zoom widget duplicated after `setContent` calls

### PR DESCRIPTION
**Pull request by handreyrc**
Jenkins execute full downstream build

### CODE REVIEW

**Comment by handreyrc** [comment]
Jenkins execute full downstream build

**Comment by handreyrc** [comment]
Jenkins retest please

**Comment by handreyrc** [comment]
Jenkins execute full downstream build

**Comment by handreyrc** [comment]
@romartin, @inodeman Could you please review this PR?

**Comment by handreyrc** [comment]
Jenkins retest please

**Comment by handreyrc** [comment]
Jenkins execute full downstream build

File: kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-common/src/test/java/org/kie/workbench/common/dmn/webapp/kogito/common/client/editor/AbstractDMNDiagramEditorTest.java [hunk 1]
```diff
@@ -482,4 +483,15 @@ protected void assertOnDiagramLoad() {
         verify(layoutHelper).applyLayout(eq(diagram), eq(layoutExecutor));
         verify(dataTypesPage).reload();
     }
+
+    @Test
+    public void testsuperOnCloseDMNOnSetContent() {
```
Comment (manstis) [inline kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-common/src/test/java/org/kie/workbench/common/dmn/webapp/kogito/common/client/editor/AbstractDMNDiagramEditorTest.java]: Please correct the method name to `testSuperOnCloseDMNOnSetContent()`.

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-kogito-runtime/src/test/java/org/kie/workbench/common/stunner/kogito/client/editor/BPMNDiagramEditorTest.java [hunk 2]
```diff
@@ -147,4 +149,15 @@ public void testMenuInitialized() {
         editor.makeMenuBar();
         assertEquals(editor.menuBarInitialized, true);
     }
+
+    @Test
+    public void testsuperOnCloseDMNOnSetContent() {
```
Comment (manstis) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-kogito-runtime/src/test/java/org/kie/workbench/common/stunner/kogito/client/editor/BPMNDiagramEditorTest.java]: Please correct the method name to `testSuperOnCloseDMNOnSetContent()`.

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-kogito-runtime/src/test/java/org/kie/workbench/common/stunner/kogito/client/editor/BPMNDiagramEditorTest.java [hunk 3]
```diff
@@ -147,4 +149,15 @@ public void testMenuInitialized() {
         editor.makeMenuBar();
         assertEquals(editor.menuBarInitialized, true);
     }
+
+    @Test
+    public void testSuperOnCloseDMNOnSetContent() {
```
Comment (manstis) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-kogito-runtime/src/test/java/org/kie/workbench/common/stunner/kogito/client/editor/BPMNDiagramEditorTest.java]: Sorry, my bad.. I incorrectly copied and pasted my comment.

For BPMN this method should obviously be `testSuperOnCloseBPMNOnSetContent()`.

**Comment by handreyrc** [comment]
@manstis, Fixed! Sorry about that, some times Ctrl + V is a nemesis.

**Comment by manstis** [comment]
@handreyrc NP.. Did you push your latest fix?

**Comment by handreyrc** [comment]
@manstis, I've just pushed! Thanks

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-kogito-runtime/src/test/java/org/kie/workbench/common/stunner/kogito/client/editor/BPMNDiagramEditorTest.java [hunk 4]
```diff
@@ -147,4 +149,15 @@ public void testMenuInitialized() {
         editor.makeMenuBar();
         assertEquals(editor.menuBarInitialized, true);
     }
+
+    @Test
+    public void testSuperOnCloseOnSetContent() {
```
Comment (manstis) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-kogito-runtime/src/test/java/org/kie/workbench/common/stunner/kogito/client/editor/BPMNDiagramEditorTest.java]: You chose to omit the `BPMN` bit (but no problem, it does not really matter).

**Comment by handreyrc** [comment]
Jenkins retest please

**Comment by handreyrc** [comment]
Jenkins execute full downstream build

**Comment by handreyrc** [comment]
Jenkins retest please

**Comment by handreyrc** [comment]
Jenkins retest please

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by handreyrc** [comment]
jenkins execute full downstream build

**Comment by handreyrc** [comment]
jenkins execute full downstream build

**Comment by handreyrc** [comment]
Jenkins retest please

**Comment by handreyrc** [comment]
jenkins execute full downstream build

**Comment by handreyrc** [comment]
jenkins execute full downstream build

**Comment by handreyrc** [comment]
Jenkins retest please

**Comment by handreyrc** [comment]
Jenkins retest please

**Comment by handreyrc** [comment]
Jenkins retest please

**Comment by handreyrc** [comment]
Jenkins retest please

**Comment by handreyrc** [comment]
jenkins execute full downstream build

**Comment by handreyrc** [comment]
Jenkins retest please

**Comment by handreyrc** [comment]
jenkins execute full downstream build

**Comment by handreyrc** [comment]
Jenkins execute full downstream build

**Comment by handreyrc** [comment]
Jenkins execute full downstream build

**Comment by romartin** [comment]
Hey @handreyrc 

Great job, it looks good to me, but before approving, could you please verifity if the following bug, reported by @lclay2  is present even after this fix please?:

`The only possible buggy thing I could find, was toggle between full screen and back to regular screen sometime I ended up with 2 zoom toolbars sometimes in DMN. It seemed to have something to do with the state of the "Exit Full Screen" button I think? `

zooms-duplicated

Please can you let me know if this is fixed or should I report it? Thanks in afvance!

**Pull request merged**